app/models/tag_implication.rb

```diff
@@ -5,6 +5,7 @@ class TagImplication < TagRelationship
 
   array_attribute :descendant_names
 
+  validates :antecedent_name, tag_name: { disable_ascii_check: true }, if: :antecedent_name_changed?
```

Comment:
Debating if we should even have this, but as it currently stands, implications disabled ascii check, so moving it here is to maintain that feature.